deployments/values/supported-matrix.yaml

```diff
@@ -62,13 +62,13 @@ matrix:
   current:
     id: current
     acs:
-      version: &acs_ga_version "23.1"
+      version: &acs_ga_version "23.2"
       pattern: *ga_hotfixes_pattern
     activemq:
       version: "5.18"
       pattern: *ga_activemq_pattern
     share:
-      version: "23.1"
+      version: "23.2"
```

Comment:
you can reuse the *acs_ga_version anchor, it's probably a miss when we introduced it